### ISSUE TITLE
Question about loss value

### ISSUE THREAD

**Issue by diaoyingyu** [Stick-To/FCOS-tensorflow#6]
How much is the loss about when the model converges？ I have trained for 69 epochs, but the loss doesn't seem to drop for a long time.
Thx!

**Comment by Stick-To** [comment]
I forgot it. 
You can test some images.
